README.md

```diff
@@ -110,18 +136,6 @@ Register it in the di-container:
 services.AddSingleton<ISettingsResolver, CustomSettingsResolver>();
 ```
 
-## Site specific settings
-*From version 3:* If "Use site specific assets" is enabled for a site, a "Site settings Root" folder is created for each site where settings for the site can be stored.
-
-![Settings Gadget](./settings_gadget.jpg)
-
-If upgrading from version 2 *AND* and sites in the solution has had site specific assets enabled.
-
-1. Upgrade to version 3.x
-2. Using the blocks gadget, locate the folder "For this site / Settings Root"
-3. Move it into "For all sites / Settings Root"
-4. Now all settings are available in the settings gadget so that they can be moved into the specific site they target (or being kept as shared)
-
 ## Removing Settings classes
```

Comment:
This section should no longer be needed? since updating the base column was fixed by Optimizely

Reply:
Removed the section about removing settings classes.